src/definitions/behaviors/form.js

```diff
@@ -136,7 +137,10 @@
                 },
 
                 attachEvents: function (selector, action) {
-                    action = action || 'submit';
```

Comment:
I actually dont like this change. It bloats the code, although it's clear what it does.
It is also inconsistent as many other places in the code still have " xxx || yyy"

Reply:
see https://github.com/fomantic/Fomantic-UI/commit/3b79c1cee555bf22ee10cf4be2e4978a97de73d6

this is to fix `unicorn/prefer-default-parameters` rule, only a few lines were required to be updated

IMHO the `if` is more readable than reassigning the same value, but that just my opinion, I did not fixed the code in general nor plan do to so

Reply:
Wouldn't it be better to remove that rule from eslint until we finally dropped IE11 support? The desired fix according to that rule would be
```
attachEvents: function (selector, action = 'submit') {
```

Reply:
In this PR, I focus on no functional change. The fix with default argument is changing the "method prototype", please confirm we can use it safely in all https://github.com/fomantic/Fomantic-UI/commit/3b79c1cee555bf22ee10cf4be2e4978a97de73d6 changes, if yes, I will change it, if not, I prefer to keep the changes as they are.